spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -236,27 +238,26 @@ This specification does not require implementations to follow a
 particular processing model. An implementation MAY use whatever
 processing model is appropriate to its environment, as long as it
 produces a running Web Service with the proper contract and runtime
-behavior. For example, one implementation might process the JSR-181
+behavior. For example, one implementation might process the Jakarta Web Services Metadata
 annotations directly within the Java compiler to generate a deployable
 Web Service as the output of compilation; another might provide tools to
 convert a compiled service implementation bean into a set of artifacts
 that can be deployed into the container; and a third might configure its
 runtime container directly off the Java source or class file. Each
-implementation is conformant with JSR-181 as long as it produces a Web
+implementation is conformant with Jakarta Web Services Metadata as long as it produces a Web
```

Comment:
Jakarta Web Services Metadata Specification

Reply:
ok and other places too (https://github.com/eclipse-ee4j/jws-api/pull/69/commits/3d54eeebc5474552c7a719f3932076709eb58551)